### ISSUE TITLE
Issues when canceling a password entry to pkexec

### ISSUE THREAD

**Issue by Fat-Zer** [lxqt/lxqt-admin#135]
Steps to reproduce:
1. Perform any action which requires pkexec action (e.g add a user)
2. Cancel on password prompt

Actual results:
1. An error message
2. The dialog with settings disappears
3. (In case of adding a new user) A password prompt for the new user is showed.

Expected results:
- Quite the opposite of actual results.

**Comment by chris2553** [comment]
in lxqt-admin-0.15, the (with my other patch applied - see issue #136), the UI freezes when the pkexec password dialogue is cancelled. I've created a patch that avoids the freeze. It also avoids the reboot messagebox popping up when the RTC update fails. This patch depends my patch for issue 136.
[lxqt-admin-0.15.0-fix-hang-when-pkexec-dialogue-cancelled.patch.zip](https://github.com/lxqt/lxqt-admin/files/4746877/lxqt-admin-0.15.0-fix-hang-when-pkexec-dialogue-cancelled.patch.zip)

**Comment by luis-pereira** [comment]
@Fat-Zer This hapens with what OS ? You skipped that part of the form.

**Comment by chris2553** [comment]
Doh! I replied to Luis rather than to the list. Here's what I said:

On 08/06/2020 19:37, Luís Pereira wrote:
> @Fat-Zer <https://github.com/Fat-Zer> This hapens with what OS ? You skipped that part of the form.
> 

Sorry, I didn't know there was a form. Anyway, my system was originally Linux From Scratch, but I have added various
things to it since I first built it 3 years ago (e.g. lxqt, rpm to build and manage packages) and I run up-to-date
kernels on it. I keep the packages up to date too, so it's a kind of rolling release, but very stable. My system does
not have systemd - I use sysvinit to run the boots scripts.
